glide.lock

```diff
@@ -25,14 +25,15 @@ imports:
   version: 888eb0692c857ec880338addf316bd662d5e630e
   subpackages:
   - proto
+  - ptypes/any
 - name: github.com/hashicorp/go-msgpack
   version: fa3f63826f7c23912c15263591e65d54d080b458
   subpackages:
   - codec
 - name: github.com/hashicorp/memberlist
   version: a93fbd426dd831f5a66db3adc6a5ffa6f44cc60a
 - name: github.com/intelsdi-x/gomit
-  version: db68f6fda248706a71980abc58e969fcd63f5ea6
+  version: 286c3ad6599724faed681cd18a9ff595b00d9f3f
 - name: github.com/intelsdi-x/snap-plugin-lib-go
   version: a8b9252d1c8305deb3c24495f1e9ce26607fcbd7
```

Comment:
Could you check the version of snap-plugin-lib-go? I suppose it should point to commit 3e667 at least (taking the latest is preferable).